### PR TITLE
Disable CSRF when auth is disabled

### PR DESCRIPTION
**Pull request by AsgerStage**
CSRF was still enabled when authentication was disabled. This was causing POST requests to fail.

### CODE REVIEW

File: src/main/java/org/dcsa/core/security/SecurityConfig.java [hunk 1]
```diff
@@ -53,13 +53,15 @@ public SecurityWebFilterChain securityWebFilterChain(ServerHttpSecurity http) {
                     .oauth2ResourceServer()
                     .jwt()
                     .jwtAuthenticationConverter(new JwtAuthenticationConverter())
-                    .and();
+                    .and()
```
Comment (HenrikHL) [inline src/main/java/org/dcsa/core/security/SecurityConfig.java:56]: Two rows with same content?!?

**Review by HenrikHL — DISMISSED** [review]

**Comment by HenrikHL** [comment]
You need to bump Core

File: src/main/java/org/dcsa/core/security/SecurityConfig.java (hunk 1 above)
Comment (AsgerStage) [reply src/main/java/org/dcsa/core/security/SecurityConfig.java:56]: Looks a bit weird, but is is needed. The first and() returns a OAuth2ResourceServerSpec object, and the second one (used on the OAuth2ResourceServerSpec) returns a ServerHttpSecurity Object, where cors can be enabled

**Review by HenrikHL — APPROVED** [review]
Tests are failing - maybe workflow file needs to be updated

**Review by HenrikHL — APPROVED** [review]

**Review by HenrikHL — APPROVED** [review]

**Pull request merged**